apps/site/assets/ts/shuttles/components/ShuttlesOverview.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { ReactElement, useState, useEffect } from "react";
+
+interface Props {
+  places: {
+    0: string;
+    1: string;
+  };
+}
+
+const ShuttlesOverview = ({ places }: Props): ReactElement<HTMLElement> => {
+  const [selectedDirectionId, setSelectedDirectionId] = useState("");
+  useEffect(() => {}, [selectedDirectionId]);
+
+  const directionButton = (
+    directionId = "",
+    headsign = "All Directions"
+  ): JSX.Element => (
+    <button
+      key={`${headsign}`}
+      type="button"
+      className={`btn btn-secondary btn-sm ${
+        selectedDirectionId === directionId ? "focus" : ""
+      }`}
+      onClick={() => setSelectedDirectionId(directionId)}
+    >
+      {headsign}
+    </button>
+  );
+
+  return (
+    <>
+      <div role="group" aria-label="Filter shuttle map by direction">
+        {directionButton()}
+        {Object.entries(places).map(([directionId, headsign]) =>
```

Comment:
Let's throw a `sort` in here, since IIRC the order of the elements of the return value of `entries` is not guaranteed. It'll _probably_ always be 0 then 1...but with one method call we can be certain.